hackathon_site/dashboard/frontend/src/slices/hardware/cartSlice.ts

```diff
@@ -26,6 +30,47 @@ export const cartReducerName = "cart";
 export const initialState = cartAdapter.getInitialState(extraState);
 export type CartState = typeof initialState;
 
+// Thunk
+interface RejectValue {
+    status: number;
+    message: any;
+}
+
+export const submitOrder = createAsyncThunk<
```

Comment:
Think you could throw together some basic unit tests for the success and rejected cases, mocking the post function? Make a store dispatch, then wait for the store to be in the expected state as with some of the other tests